board/app/controllers/user_controller.php

```diff
@@ -1,92 +1,129 @@
 <?php
 
-class UserController extends AppController{
+class UserController extends AppController 
+{
+
+const LOGIN_PAGE = 'login';
+const HOME_PAGE = 'home';
+const REGISTER = 'register';
+const REGISTER_END = 'register_end';
+const EDIT = 'edit';
+const EDIT_END = 'edit_end';
 
     public function register() 
     {
-        if (is_logged_in()) 
-        {
-            redirect(url('user/home'));
+        if (is_logged_in()) {
+            redirect(url('comment/home'));
```

Comment:
Removed po. Thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/9dbba76440ddb379099d460816478f537867af7d